app.js

```diff
@@ -45,17 +47,22 @@ if (program.address) {
 }
 
 if (program.peers) {
-	if (typeof program.peers === 'string') {
-		appConfig.peers.list = program.peers.split(',').map(function (peer) {
-			peer = peer.split(':');
+	appConfig.peers.list = parametersReader.convertToAddressList(program.peers, appConfig.port);
+}
+
+if (program.sync) {
+	appConfig.syncPeers = {
+		list: parametersReader.convertToAddressList(program.sync, appConfig.port).map(function (syncPeer) {
```

Comment:
these fields aren't needed in case of reading peers list. `convertToAddressList` just reads from console input and turns into json